public/javascripts/SVLabel/src/SVLabel/mission/Mission.js

```diff
@@ -153,6 +153,10 @@ function Mission(parameters) {
             };
             svl.labelCounter.reset();
         }
+
+        if (svl.main.isAnAnonymousUser() && !svl.onboarding.isOnboarding()){
```

Comment:
I don't encounter this error when I run the branch locally. I'm not sure why the error would occur, the line uses existing svl functions.